### ISSUE TITLE
Empty breadcrumb <ol> tags being rendered

### ISSUE THREAD

**Issue by corylown** [projectblacklight/arclight#1616]
`<ol>` tags must contain at least one child `<li>` tag to be valid, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/ol#technical_summary

This is a markup and accessibility issue.

In the group by view we look an additional level deep for breadcrumbs, avoiding rendering the parent as a breadcrumb in this view. Since we're not checking whether there are any breadcrumbs to render we are sometimes rendering empty `<ol>` tags where the breadcrumbs would be.

<img width="875" height="599" alt="Image" src="https://github.com/user-attachments/assets/491f2b6c-ae72-4b07-878a-feabbc8dab69" />

<img width="813" height="218" alt="Image" src="https://github.com/user-attachments/assets/11f1247a-1882-42cc-b3f7-4d5f05c4465c" />
